src/test/bin/winkernel/control.cpp

```diff
@@ -1167,6 +1167,10 @@ QuicTestCtlEvtIoDeviceControl(
         QuicTestCtlRun(QuicTestStreamAbortConnFlowControl());
         break;
 
+    case IOCTL_QUIC_RUN_STREAM_RELIABLE_RESET:
```

Comment:
Always put the new case statement at the end of the others.

Reply:
Also, you'll need to update the big array before this function.

Reply:
done. will remember to always put new cases at the end.